src/styles/notifications.less

```diff
@@ -263,4 +263,112 @@ ux-alert {
         background-color: @alert-dark-bg;
         border: 1px solid @alert-dark-border;
     }
-}
\ No newline at end of file
+}
+
+/*
+ * Badge
+ */
+
+.ux-badge-container {
+    position: relative;
+    overflow: visible;
+
+    .ux-badge {
```

Comment:
Looking at the upcoming badge-inline component, you might consider un-nesting this style rule along with some others below so that they can be used separately.

Reply:
Saw that this was resolved, did you forget to make the change?